Client/Frontend/Browser/Onboarding/OnboardingRewardsAgreementView.swift

```diff
@@ -0,0 +1,231 @@
+            $0.backgroundBehavior = .pauseAndRestore
+            $0.loopMode = .loop
+            $0.play()
+        }
+        
+        private let descriptionView = UIView().then {
+            $0.layer.cornerRadius = 12
+            $0.layer.maskedCorners = [.layerMinXMinYCorner, .layerMaxXMinYCorner]
+        }
+        
+        private let descriptionStackView = UIStackView().then {
+            $0.axis = .vertical
+            $0.spacing = 32
+        }
+        
+        private let descriptionCheckbox = UIButton().then {
+            $0.setImage(#imageLiteral(resourceName: "checkbox_off"), for: .normal)
+            $0.setImage(#imageLiteral(resourceName: "checkbox_on"), for: .selected)
+            $0.setImage(#imageLiteral(resourceName: "checkbox_on"), for: .highlighted)
+            $0.adjustsImageWhenHighlighted = true
+            
+            $0.contentMode = .scaleAspectFit
+            $0.setContentHuggingPriority(.required, for: .horizontal)
+            $0.setContentCompressionResistancePriority(.required, for: .horizontal)
+        }
+        
+        private let titleLabel = CommonViews.primaryText(Strings.OBRewardsAgreementTitle).then {
+            $0.numberOfLines = 0
+        }
+        
+        private lazy var descriptionLabel = UITextView().then {
+            $0.delaysContentTouches = false
+            $0.isEditable = false
+            $0.isScrollEnabled = false
+            $0.isSelectable = true
+            $0.backgroundColor = .clear
+            $0.textDragInteraction?.isEnabled = false
+            $0.textContainerInset = .zero
+            $0.font = UIFont.systemFont(ofSize: 16, weight: UIFont.Weight.regular)
+            $0.delegate = self
+            
+            $0.linkTextAttributes = [
+              .font: $0.font!,
+              .foregroundColor: UX.linkColor,
+              .underlineStyle: 0
+            ]
+            
+            $0.textAlignment = .center
+        }
+        
+        private lazy var textStackView = UIStackView().then { stackView in
+            stackView.axis = .vertical
+            stackView.spacing = 8
+            let descriptionStackView =  UIStackView(arrangedSubviews: [descriptionLabel, descriptionCheckbox]).then {
+                $0.alignment = .center
+                $0.spacing = 65.0
+            }
+            
+            [titleLabel, descriptionStackView].forEach(stackView.addArrangedSubview(_:))
+        }
+        
+        private let buttonsStackView = UIStackView().then {
+            $0.distribution = .equalCentering
+        }
+        
+        private func updateDescriptionLabel() {
+            descriptionLabel.attributedText = {
+                let text = NSMutableAttributedString(string: Strings.OBRewardsAgreementDetail, attributes: [
+                    .font: UIFont.systemFont(ofSize: 16, weight: UIFont.Weight.regular),
+                    .foregroundColor: titleLabel.textColor!
```

Comment:
we could unwrap it at the beginning of the method, it's used in few places here

or make something with a fallback like `let color = titleLabel.textColor : .black`